tests/vectors/protocol-definitions/dex.json

```diff
@@ -1,15 +1,18 @@
 {
-  "labels": {
-    "ask": {
+  "recordTypes": [
+    {
+      "id": "ask",
       "schema": "https://tbd/website/tbdex/ask"
     },
-    "offer": {
+    {
+      "id": "offer",
       "schema": "https://tbd/website/tbdex/offer"
     },
-    "fulfillment": {
+    {
+      "id": "fulfillment",
       "schema": "https://tbd/website/tbdex/fulfillment"
     }
-  },
+  ],
   "records": {
     "ask": {
```

Comment:
Sanity: will there be a follow up PR that removes the dynamic properties under `records` (e.g. `ask` in this case)?

Reply:
Yes, that's the plan. I didn't want to put it all in one PR